Gemfile.lock

```diff
@@ -517,6 +529,7 @@ DEPENDENCIES
   pry-nav
   quality-measure-engine!
   rails (~> 4.2.7)
+  rails_best_practices
```

Comment:
It looks like you removed this from the `Gemfile`, but didn't rerun `bundle install`

Reply:
Whoops. My b. Updated now.